android/app/src/main/java/org/pathcheck/covidsafepaths/util/COVIDSafePathsConfig.kt

```diff
@@ -0,0 +1,23 @@
+package org.pathcheck.covidsafepaths.util
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+import org.pathcheck.covidsafepaths.BuildConfig
+
+class COVIDSafePathsConfig(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {
+
+    override fun getName(): String {
```

Comment:
minor thing, but not sure we are ever going to need this function?

Reply:
good question: it's used to expose the name of the module to the `JS` layer, so we are stuck with it I believe